starport/interface/cli/starport/cmd/network_chain_create.go

```diff
@@ -8,51 +8,61 @@ import (
 
 	"github.com/manifoldco/promptui"
 	"github.com/spf13/cobra"
+	"github.com/tendermint/starport/starport/pkg/cliquiz"
 	"github.com/tendermint/starport/starport/pkg/clispinner"
 	"github.com/tendermint/starport/starport/pkg/events"
 	"github.com/tendermint/starport/starport/pkg/xurl"
 	"github.com/tendermint/starport/starport/services/networkbuilder"
 )
 
+const (
+	flagChain  = "chain"
+	flagSource = "source"
+)
+
 func NewNetworkChainCreate() *cobra.Command {
 	c := &cobra.Command{
-		Use:   "create [chainID] [repo]",
+		Use:   "create",
 		Short: "Create a new network",
 		RunE:  networkChainCreateHandler,
-		Args:  cobra.ExactArgs(2),
 	}
+	c.Flags().String(flagChain, "", "Chain's id")
```

Comment:
```suggestion
	c.Flags().String(flagChain, "", "Chain ID")
```
I think it's ok to write it like this

Reply:
solved by e8429383da6a83e0ec0aba10e358390a68f40a86